server/pogo/lobby_messages.go

```diff
@@ -9,3 +9,9 @@ type LobbyMsg struct {
 	ID      string   `json:"id"`
 	Players []string `json:"players"`
 }
+
+type ReadyMsg struct {
```

Comment:
Does it make sense to just use the `LobbyMsg` for all lobby-related updates? I don't see any reason we'd need a second message type as we can just include all player "readiness" information on the `LobbyMsg`.